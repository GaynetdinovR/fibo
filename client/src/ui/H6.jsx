import styles from "../styles/Ui.module.sass";
import classNames from "classnames";

const H6 = ({children, className}) => {
	return (
        <h6 className={classNames(className, styles.h6)}>
            {children}
        </h6>
	);
};

export default H6;