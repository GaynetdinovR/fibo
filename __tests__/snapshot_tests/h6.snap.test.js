import renderer from "react-test-renderer";
import { it, expect } from "@jest/globals";
import H6 from "../../src/ui/H6.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<H6>H6</H6>).toJSON();
	expect(tree).toMatchSnapshot();
});
