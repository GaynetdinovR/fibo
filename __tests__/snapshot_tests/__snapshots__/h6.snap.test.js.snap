// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h6
  className="h6"
>
  H6
</h6>
`;
